lib/elasticsearch/elastic-query-builder.js

```diff
@@ -343,9 +342,13 @@ class ElasticQueryBuilder {
   /**
   * Boost bibs with strong subjectLiteral matches
   **/
-  boostSubjectMatches () {
-    const q = this.request.params.subject_prefix
+  boostSubjectMatches (q = null) {
+    q = q || this.request.params.subject_prefix
     this.query.addShoulds([
+      phraseMatch('subjectLiteral.raw', q, 50),
+      phraseMatch('parallelSubjectLiteral.raw', q, 50),
```

Comment:
I don't think we want to promote phrase-matches when a `?subject_prefix=` is used. Use of `subject_prefix=` indicates only complete/prefix matches should be returned. Looks like `boostSubjectMatches` was previously used for prefix matching, but if `boostSubjectMatches` is going to now include phrase-matching, I think it's time to separate these boosting functions. I'd just copy the two `termMatch` clauses below into the `applySubjectPrefix` function and have `applySubjectPrefix` no longer call `boostSubjectMatches`.

That means also that you don't need to do the parameter logic you have above with optional `q` param; `boostSubjectMatches` doesn't need to take any param and can just trust the query to be in `this.request.querySansQuotes()` like other boosting functions